{"name": "Тест на ошибку при авторизации с неправильным полем Пароль", "status": "passed", "steps": [{"name": "Создаём пользователя", "status": "passed", "start": 1747663845209, "stop": 1747663845209}, {"name": "Меняем тело авторизации(вставляем в поле Пароль неправильное значение)", "status": "passed", "start": 1747663845209, "stop": 1747663845956}], "start": 1747663845209, "stop": 1747663845956, "uuid": "4d6700fa-724a-4160-8804-3c868f0f51f3", "historyId": "fe47d3b922a866d32f6eab8f1de4330c", "testCaseId": "fe47d3b922a866d32f6eab8f1de4330c", "fullName": "tests.test_user_login.TestUserLogin#test_wrong_password_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "DESKTOP-0807K0Q"}, {"name": "thread", "value": "40372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}